src/psyclone/gocean1p0.py

```diff
@@ -729,10 +706,10 @@ def add_bounds(bound_info):
             try:
                 _ = Nonlabel_Do_Stmt(do_string)
             except NoMatchError as err:
-                raise ConfigurationError("Expression '{0}' is not a "
+                six.raise_from(ConfigurationError("Expression '{0}' is not a "
                                          "valid do loop boundary. Error "
```

Comment:
pycodestyle complains about the indenting here.

Reply:
For me, too - and I have *absolutely* no idea how I could have missed that. I mean, I did run a script that runs pycodestyle and pylint for me, since I have separate commits to fix some stuff :(